Python 3.7.6 (tags/v3.7.6:43364a7ae0, Dec 19 2019, 00:42:30) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
>>> 
========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========== RESTART: F:/15th_june_AI_batch/day_6/sikh.py =========
>>> abc()
<__main__.abc object at 0x000001A6EE763088>
>>> x = abc()
>>> x.fun()
Traceback (most recent call last):
  File "<pyshell#2>", line 1, in <module>
    x.fun()
AttributeError: 'abc' object has no attribute 'fun'
>>> x.func()
hello
>>> x.a
10
>>> x.func(u)
Traceback (most recent call last):
  File "<pyshell#5>", line 1, in <module>
    x.func(u)
NameError: name 'u' is not defined
>>> x.func()
hello
>>> x.func(8)
Traceback (most recent call last):
  File "<pyshell#7>", line 1, in <module>
    x.func(8)
TypeError: func() takes 1 positional argument but 2 were given
>>> x.func(8)
Traceback (most recent call last):
  File "<pyshell#8>", line 1, in <module>
    x.func(8)
TypeError: func() takes 1 positional argument but 2 were given
>>> x.func()
hello
>>> x.func(8)
Traceback (most recent call last):
  File "<pyshell#10>", line 1, in <module>
    x.func(8)
TypeError: func() takes 1 positional argument but 2 were given
>>> 
========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
>>> 
========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
Traceback (most recent call last):
  File "F:/15th_june_AI_batch/day_6/editor.py", line 30, in <module>
    tk.Label(app, text = "bye").pack(x = 130 , y = 80)
  File "C:\Users\Shalu Kumari\AppData\Local\Programs\Python\Python37\lib\tkinter\__init__.py", line 2143, in pack_configure
    + self._options(cnf, kw))
_tkinter.TclError: bad option "-x": must be -after, -anchor, -before, -expand, -fill, -in, -ipadx, -ipady, -padx, -pady, or -side
>>> 
========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\Shalu Kumari\AppData\Local\Programs\Python\Python37\lib\tkinter\__init__.py", line 1705, in __call__
    return self.func(*args)
TypeError: operate() missing 1 required positional argument: 'e'
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\Shalu Kumari\AppData\Local\Programs\Python\Python37\lib\tkinter\__init__.py", line 1705, in __call__
    return self.func(*args)
TypeError: operate() missing 1 required positional argument: 'e'

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi
hi

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
>>> 
========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
1+2

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========

========= RESTART: F:/15th_june_AI_batch/day_6/editor.py ========
>>> 